vectorize.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var path = require('path')
+var argv = require('minimist')(process.argv.slice(2))
+var polyspine = require('polyspine')
+var tilebelt = require('tilebelt')
+var flatten = require('geojson-flatten')
+var normalize = require('geojson-normalize')
+var _ = require('lodash')
+var distance = require('turf-line-distance')
+var simplify = require('turf-simplify')
+
+var input = JSON.parse(fs.readFileSync(path.join(__dirname, argv._[0])))
+input = normalize(input)
+input.features = _.flatten(input.features.map(f => flatten(f)))
+
+// arguments given as filename, then z y x
+var tile = argv._.slice(1, 4).reverse()
```

Comment:
Could we go with either `x y z` or `z x y`?  (Slight personal pref for the latter, but only slight)